### DISCUSSION TITLE
Galaxy Brain Achievement

### DISCUSSION THREAD

**Discussion by drknzz** [drknzz/GitHub-Achievements#13] [Q&A] [answered]
❗

**Comment by mgks** [comment]
I think the badge stays even after the discussion is unmarked as an answer later. Let's give it a try.

**Reply by drknzz** [reply]
Yeah, it does stay 👍

**Reply by lironmiz** [reply]
how i get the badge?

**Comment by YSD98** [comment, marked as answer]
I want a galaxy brain Badge
